File: dataset/output-bk/OPENAI_gpt-4o/Task157_OPENAI_gpt-4o.java
public class MemoryAllocation {
    public static void main(String[] args) {
        // Allocate memory for an integer
        Integer myVar = new Integer(42);
        System.out.println("Java: Value = " + myVar);

        // Deallocate memory
        myVar = null;
        System.gc(); // Request garbage collection
    }
}

File: dataset/output/OPENAI_gpt-4o/Task157_OPENAI_gpt-4o.cpp
#include <iostream>

int main() {
    // Allocate memory for an integer
    int* int_ptr = new int(10);
    std::cout << "Value at allocated memory: " << *int_ptr << std::endl;

    // Deallocate memory
    delete int_ptr;

    return 0;
}